momento-sdk/src/main/java/momento/sdk/UserHeaderInterceptor.java

```diff
@@ -13,16 +13,21 @@
 final class UserHeaderInterceptor implements ClientInterceptor {
 
   private static final Metadata.Key<String> AUTH_HEADER_KEY =
-      Metadata.Key.of("Authorization", ASCII_STRING_MARSHALLER);
+      Metadata.Key.of("authorization", ASCII_STRING_MARSHALLER);
   private static final Metadata.Key<String> SDK_AGENT_KEY =
-      Metadata.Key.of("Agent", ASCII_STRING_MARSHALLER);
+      Metadata.Key.of("agent", ASCII_STRING_MARSHALLER);
+  private static final Metadata.Key<String> RUNTIME_VERSION_KEY =
+      Metadata.Key.of("runtime-version", ASCII_STRING_MARSHALLER);
   private final String tokenValue;
-  private final String sdkVersion =
-      String.format("java:%s", this.getClass().getPackage().getImplementationVersion());
-  private static volatile boolean isUserAgentSent = false;
+  private final String sdkVersion;
+  String runtimeVer = System.getProperty("java.vendor") + ", " + System.getProperty("java.version");
```

Comment:
Can you make this private final to match the sdkVersion? And define them both in the same place.